### ISSUE TITLE
Why isn't the action published to Github market place?

### ISSUE THREAD

**Issue by suryasr007** [unsplash/comment-on-pr#22]

**Comment by aaronklaassen** [comment]
Because this Action was initially just in the Actions beta and I'm not sure the marketplace was open yet when I created this? And then I never got around to it? ¯\_(ツ)_/¯

But you're right, of course. I'll publish it.

**Comment by lmeyerov** [comment]
@aaronklaassen In particular, would appreciate a versioning label to avoid regression risks between upgrades :)

**Comment by svenluijten** [comment]
Yeah the reason seems to be that there's just some "tags", and no actual GitHub "releases": https://github.com/unsplash/comment-on-pr/releases. Once there's (for example) a `v1.2.0` release, we can use this Action with `unsplash/comment-on-pr@v1` instead of having to specify the exact version (`unsplash/comment-on-pr@v1`).

@aaronklaassen I'd love to get this done; is there any way I can help? 😄

**Comment by airtonix** [comment]
re: versioning. you can just use the commit hashes (it's just a commitish reference)

**Comment by lmeyerov** [comment]
We prefer frozen published (non-repo) versions to decrease risks like someone maliciously taking it down, force-pushing something else at the version, etc...
